source/val/validation_state.cpp

```diff
@@ -447,6 +447,13 @@ uint32_t ValidationState_t::getIdBound() const { return id_bound_; }
 
 void ValidationState_t::setIdBound(const uint32_t bound) { id_bound_ = bound; }
 
+void ValidationState_t::RegisterSpirvVersion(uint32_t version) {
+  spirv_version_ = version;
+  if (spirv_version_ >= 0x10300) {
```

Comment:
That's the file version. We should make validation decisions based on the validator version.

I have an alternative fix:
https://github.com/KhronosGroup/SPIRV-Tools/pull/1429/files#diff-bdc10ce1ea59762351b41a27d6c1e2d0R176

Reply:
Ok.  Makes sense. 
It points out that we've got a hole:  When validating for SPIR-V 1.0, we should reject a module that has SPIR-V 1.3 as its version.  I'll file an issue for that.